### ISSUE TITLE
Add support for additional operators

### ISSUE THREAD

**Issue by sdurbin1** [jeffshaver/safe-app#194]
Add support for the following operators in the filters for the search page, and the filters for dashboards:

IN 
IN_MATCHES
IS_NOT_NULL
IS_NULL
LUCENE
MATCHES
NOT_EQUAL
NOT_IN
NOT_IN_MATCHES
NOT_MATCHES

**Comment by sdurbin1** [comment]
Also, for IS_NOT_NULL and IS_NULL, the user will not specify a "filter criteria". Currently when no "Filter criteria" is specified, the filters array passed to REST is empty. We should agree on a way to pass back filters for IS_NULL and IS_NOT_NULL

**Comment by jeffshaver** [comment]
@sdurbin1 yeah. we will have to make special cases for those since we shouldn't strip off that filter. `IS_NULL` and `IS_NOT_NULL` don't need associated values.

**Comment by knorris** [comment]
It was agreed to just add '!=' for now.
